### DISCUSSION TITLE
YI Home 1080p: I'm not sure about firmware and backup

### DISCUSSION THREAD

**Discussion by Jaanzter** [roleoroleo/yi-hack-Allwinner-v2#751] [General]
Hi

I tried to match a serial number and current firmware version, but couldn't make any sense/find right firmware to upload my camera.
Also I tried to backup current firmware, but nothing happened. [this](https://github.com/roleoroleo/yi-hack-Allwinner/wiki/Dump-your-backup-firmware-(SD-card))
I thought it was this Yi 1080p Home 6FUS | 2.1.0* | y23, but nothing happened when tried to upload this. [from here](https://github.com/roleoroleo/yi-hack-MStar#supported-cameras)
Only thing matched, was a firmware version.

Camera recognizes my 32Gb sdcard 

I would appreciate if someone kindly helps me little.

![YI Home current firmware](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/82608937/369959e7-573b-42f6-986f-dafe0dad3046)
![YI Home serial number](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/82608937/c56c8e57-1431-484a-847f-2d05960b7cfc)

**Comment by roleoroleo** [comment]
It should be a MStar y23.
There is no backup procedure for MStar, but the installation with this https://github.com/roleoroleo/yi-hack-MStar/releases
should work

**Reply by Jaanzter** [reply]
Thanks, I'll try.

Edit:
Downloaded y23_0.5.3.tgz and extracted files (home_y23 and sys_y23) to root of the sdcard (32Gb).
Inserted it to the camera, boot it up. Yellow light lit up for few seconds and blue light started to flash (wifi I believe) and
now blue light is constantly on)

Camera recognizes the sdcard just fine.

**Reply by roleoroleo** [reply]
And what about the hack? Is it loaded?

**Reply by Jaanzter** [reply]
Cannot connect to ip address I found from app.
So, I believe it's not loaded.

> ERR_CONNECTION_REFUSED

**Reply by roleoroleo** [reply]
Try a telnet connection.

**Reply by Jaanzter** [reply]
Thanks for helping, you must have better/more important things to do too.

Putty and port 23 & 21
Network error: Connection refused.

Asus wifi router sees it too.
Does it help if you see the log files that camera created?

![ASUS Wireless Router RT-AX86U](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/82608937/0b6be4a9-c544-48d5-b593-19dc95e34fdc)
--------
![NVIDIA GeForce Overlay DT](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/82608937/b3ca1a31-cb08-4229-9e55-849d51ad6062)
--------
![2023-11-19 18_42_36-NVIDIA GeForce Overlay DT](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/82608937/40b27708-813f-4030-ae40-4c1309df9f45)

**Reply by roleoroleo** [reply]
The only thing I can think of is that the sd card is not properly recognized by the bootloader.
Please, try another card.

**Reply by Jaanzter** [reply]
That will be tomorrow then. All other sdcards are 64+Gb.

**Reply by Jaanzter** [reply]
Yup, same thing.
I bought a 16Gb sdcard and camera acts same way: yellow light stays on few seconds, then blue light starts flashing and after that it stays on too.

Gosh Darnit, can't even brick my camera. :D 

Factory reset and nothing. Tried http and telnet.

**Reply by roleoroleo** [reply]
I don't know exactly what model is the cam.
Please, share the log files. I will try to find something.

> ![2023-11-19 18_42_36-NVIDIA GeForce Overlay DT](https://user-images.githubusercontent.com/82608937/284075565-40b27708-813f-4030-ae40-4c1309df9f45.png)

**Reply by Jaanzter** [reply]
I would feel more comfortable if I could provide those files privately or pw protect them.
email maybe?

**Reply by Jaanzter** [reply]
if you don't want to show yours here, mine is REMOVED

**Reply by Jaanzter** [reply]
sent

**Reply by roleoroleo** [reply]
Sorry but this cam is not supported by this hack.
It's based on hi3518 chip.
You should try yi-hack-v5

**Reply by Jaanzter** [reply]
Yay!! It works, thanks @roleoroleo
